deploy/ansible/roles/goshimmer-node/templates/docker-compose-entrynode.yml.j2

```diff
@@ -0,0 +1,25 @@
+#jinja2: trim_blocks:True, lstrip_blocks:True
+version: "3"
+
+services:
+  entrynode:
+    image: iotaledger/goshimmer:latest
+    container_name: entrynode
+    restart: unless-stopped
+    network_mode: "host"
+    volumes:
+      - "./db:/db"
+      - "/etc/localtime:/etc/localtime:ro"
+    environment:
+      - DATABASE_DIRECTORY=/db/mainnetdb
+      - AUTOPEERING_SEED={{ entryNodeSeed }}
+      - PROFILING_BINDADDRESS=0.0.0.0:6060
+    entrypoint: /run/goshimmer
+    command: >
+      --skip-config=true
+      --autopeering.port=15626
+      --autopeering.networkVersion=8000
```

Comment:
You want a random network version to not mix with nodes from the previous deployment? 

The problem with using a random number is how do you know that number in the future when you want to connect to the network from the local machine e.g. @capossele is there a way to request the network version from a node by web API? 

BTW, we could use the SHA of the commit instead of a random number for the network version.